src/storage/exec/GetNeighborsNode.h

```diff
@@ -138,6 +141,24 @@ class GetNeighborsNode : public QueryNode<VertexID> {
     return nebula::cpp2::ErrorCode::SUCCEEDED;
   }
 
+  bool isDuplicatedSelfReflectiveEdge(const folly::StringPiece& key) {
+    std::string srcID = NebulaKeyUtils::getSrcId(context_->vIdLen(), key).str();
```

Comment:
Better use `folly::StringPiece srcID = NebulaKeyUtils::getSrcId` or just `auto` here to avoid extra string copy here. If it is a self reflective edge, then convert it to string when concatenating `localKey`.

Reply:
good advice. fixed.